openvisualizer/moteProbe/moteProbe.py

```diff
@@ -21,18 +21,51 @@
 import time
 import sys
 
+import paho.mqtt.client as mqtt
+import json
+import Queue
+
 from   pydispatch import dispatcher
 import OpenHdlc
 import openvisualizer.openvisualizer_utils as u
 from   openvisualizer.moteConnector import OpenParser
 from   openvisualizer.moteConnector.SerialTester import SerialTester
 
-#============================ functions =======================================
+#============================ defines =========================================
+
+BROKER_ADDRESS              = "argus.paris.inria.fr"
+NOTIFY_SERIALBYTES_TOPIC    = 'opentestbed/deviceType/mote/deviceId/+/notif/frommoteserialbytes'
+TOTAL_NUMBER_MOTES_TESTBED  = 76
```

Comment:
change logic: in `findSerialPorts`, issue a `opentestbed/deviceType/box/deviceId/all/cmd/status` command and wait 10s for all responses.